scripts/validation/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "validation",
+  "version": "1.0.0",
+  "description": "for validation of embedded examples in asyncapi.md file",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "validate:examples": "node embedded-examples-validation.js"
+  },
+  "author": "Animesh Kumar",
+  "license": "ISC",
```

Comment:
Any particular reason why ISC instead of Apache 2.0 ?

Reply:
Don't have much idea about licensing, will change it to the recommended one...

Reply:
resolved through e3d6a78